Makefile

```diff
@@ -215,5 +215,5 @@ clean-html:
 _work/venv/.stamp: docs/requirements.txt
 	rm -rf ${@D}
 	python3 -m venv ${@D}
-	. ${@D}/bin/activate && pip install -r $<
+	. ${@D}/bin/activate && pip install wheel && pip install -r $<
```

Comment:
should it be in `requirements.txt`

Reply:
I understood it so some packages in `requirements.txt` depend on wheel but don't declare the dependency. As result placing it in `requirements.txt` doesn't help to get a correct install sequence. If it's the case then @hj-johannes-lee has a chance to contribute to the packages.

Reply:
@hj-johannes-lee is it the case? Did you try to add wheel to the requirements.txt ?

Reply:
@mythi @rojkov @bart0sh As I mentioned, I have tried to solve errors by putting 'wheel' in requirements.txt but it doesn't work.